pkg/querier/tripperware/instantquery/instant_query.go

```diff
@@ -49,9 +55,10 @@ func NewInstantQueryCodec(compressionStr string, defaultCodecTypeStr string) ins
 	}
 
 	return instantQueryCodec{
-		compression:      compression,
-		defaultCodecType: defaultCodecType,
-		now:              time.Now,
+		compression:              compression,
+		defaultCodecType:         defaultCodecType,
+		now:                      time.Now,
+		rulerQueryResponseFormat: rulerQueryResponseFormat,
```

Comment:
What kind of rule query response format should be configured at Ruler and Query Frontend gets what format Ruler asks for.